### ISSUE TITLE
ansible-vault support?

### ISSUE THREAD

**Issue by deknos** [getsops/sops#585]
Hi,
is sops able to support and encrypt/decrypt ansible-vault files? That would be a tremendous help!

**Comment by ajvb** [comment]
No, sorry. Sops will only support encryption and decryption of the sops file format. If you'd like to explain your use case I'm happy to offer suggestions though.

**Comment by oxr463** [comment]
@ajvb one use case is working with Flux GitRepositories per https://github.com/fluxcd/kustomize-controller/issues/827

Being able to encrypt and decrypt with `ansible-vault` or `sops` would allow the VMware team to collaborate with the Kubernetes team more efficiently, since the previous team is using Ansible on their infrastructure while the other team is using Flux.

**Comment by felixfontein** [comment]
@oxr463 you can also use sops with Ansible, see the [community.sops](https://galaxy.ansible.com/community/sops) collection.
